app/models/event.rb

```diff
@@ -64,12 +66,11 @@ def respondents
   end
 
   def eligible_people
-    Person.active.all # In the future, this will need to honor department
-    # def eligible_people
-    #   self.departments.each do |department|
-    #
-    #   end
-    # end
+    eligible_people = []
```

Comment:
It would be very hard from the perspective of the event, as you have to "hop" across two tables - the new join table and then from department to person. And I think rails doesn't give you the handy `has_many :through` construct when that's the case.